### DISCUSSION TITLE
Need to enable Win32 (x86) in Visual Studio version

### DISCUSSION THREAD

**Discussion by briggs-w** [assimp/assimp#5028] [Q&A]
Hi,
I've used cmake to create the Visual Studio 2022 solution on Windows 10; its Configurations don't include Win32. This is a problem because I'm using assimp with several other libraries all of which use Win32, not x64. I've verified in CMakeLists.txt that WIN32 and MSVC are both true. Is there a way I can set it to allow Win32/x86 without rewriting the .vcxproj's myself?
Tx.

**Comment by turol** [comment]
You need to specify the appropriate platform (or generator?) when running cmake.
